### PR TITLE
supportutils plugin for Rancher Setup

### PR DESCRIPTION
**Pull request by bear454**
```
  Supportconfig Plugins:                       2
    Plugin: suse_rancher_setup...              Done
```

### CODE REVIEW

File: man/man8/suse_rancher_setup.8 [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+.SH FILES
+.I /usr/lib/supportconfig/plugins/suse_rancher_setup
+.RS
+
+The SUSE Rancher Setup plugin for supportutils
+.RE
+.SH AUTHOR
+James Mason <jmason@suse.com>
+Robert Schweikert <rjschwei@suse.com>
```
Comment (rjschwei) [inline man/man8/suse_rancher_setup.8]: Please set this to the team and use our public e-mail `public-cloud-dev@susecloud.net`

File: supportconfig/plugins/suse_rancher_setup [hunk 2]
```diff
@@ -0,0 +1,98 @@
+#!/bin/bash
+
+# Supportconfig Plugin for SUSE Rancher Setup
+#
+# Copyright (c) 2022 SUSE LLC, James Mason <jmason@suse.com>,
+# Robert Schweikert <rjschwei@suse.com>
```
Comment (rjschwei) [inline supportconfig/plugins/suse_rancher_setup]: Copyright is only the company, per policy. We still have some old examples around that never got cleaned up.

File: man/man8/suse_rancher_setup.8 [hunk 3]
```diff
@@ -0,0 +1,26 @@
+.TH suse-rancher-setup-plugin "8" "07 Dec 2022" "suse-rancher-setup-plugin" "Support Utilities Manual"
+.SH NAME
+suse-rancher-setup-plugin \- SUSE Rancher Setup Plugin for
+.BR supportconfig (8)
+.
+.SH DESCRIPTION
+Supportconfig plugins add functionality to the supportconfig script. This
+plugin extends supportconfig's functionality to include pertinent information
+for instances running SUSE Rancher Setup. Supportconfig saves the plugin output
+in the public_cloud subdirectory.
```
Comment (rjschwei) [inline man/man8/suse_rancher_setup.8]: s/public_cloud/rancher_setup/

File: supportutils-plugin-suse-rancher-setup.spec [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+
+Name:           supportutils-plugin-suse-rancher-setup
+Version:        0.1.0
+Release:        0
+Summary:        SUSE Rancher Setup plugin for supportconfig
+License:        GPL-2.0+
+Group:          System/Monitoring
+Url:            https://github.com/SUSE-Enceladus/supportutils-plugin-suse-rancher-setup
+Source0:        %{name}-%{version}.tar.bz2
+Requires:       supportutils
```
Comment (rjschwei) [inline supportutils-plugin-suse-rancher-setup.spec:27]: %ifarch %ix86 x86_64
Requires:       dmidecode
%endif

**Review by rjschwei — CHANGES_REQUESTED** [review]

File: supportutils-plugin-suse-rancher-setup.spec (hunk 4 above)
Comment (bear454) [reply supportutils-plugin-suse-rancher-setup.spec:27]: Added in [c607f00](https://github.com/SUSE-Enceladus/supportutils-plugin-suse-rancher-setup/pull/1/commits/c607f005b6a1576ba2d1dfa6ca76a092aea5ac99)

File: man/man8/suse_rancher_setup.8 (hunk 3 above)
Comment (bear454) [reply man/man8/suse_rancher_setup.8]: Fixed in [c607f00](https://github.com/SUSE-Enceladus/supportutils-plugin-suse-rancher-setup/pull/1/commits/c607f005b6a1576ba2d1dfa6ca76a092aea5ac99)

File: supportconfig/plugins/suse_rancher_setup (hunk 2 above)
Comment (bear454) [reply supportconfig/plugins/suse_rancher_setup]: 👍 Fixed in [c607f00](https://github.com/SUSE-Enceladus/supportutils-plugin-suse-rancher-setup/pull/1/commits/c607f005b6a1576ba2d1dfa6ca76a092aea5ac99)

File: man/man8/suse_rancher_setup.8 (hunk 1 above)
Comment (bear454) [reply man/man8/suse_rancher_setup.8]: Fixed in [c607f00](https://github.com/SUSE-Enceladus/supportutils-plugin-suse-rancher-setup/pull/1/commits/c607f005b6a1576ba2d1dfa6ca76a092aea5ac99)

**Review by rjschwei — APPROVED** [review]

**Pull request merged**